build-system/server/new-server/transforms/cdn/cdn-transform.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```

Comment:
This was modified and renamed from `script/script-transform.ts` hence 2020. I can update the year if desired.

Reply:
👍